server/__init__.py

```diff
@@ -44,30 +44,38 @@ def createMiddleWare(app, exporter):
     return middleware
 
 
-def register_routes_common(app):
+def register_routes_base_dc(app):
     # apply the blueprints for all apps
+    from routes import (protein, redirects, special_announcement)
+    app.register_blueprint(protein.bp)
+    app.register_blueprint(redirects.bp)
+    app.register_blueprint(special_announcement.bp)
+    from routes.api import (protein as protein_api)
+    app.register_blueprint(protein_api.bp)
+
+
+def register_routes_private_dc(app):
+    ## apply the blueprints for private dc instances
     pass
 
 
-def register_routes_main_app(app):
-    # apply the blueprints for main and private app
-    from routes import (protein, browser, dev, factcheck, place, placelist,
-                        ranking, redirects, search, static, tools, topic_page)
+def register_routes_common(app):
+    # apply the blueprints for main app
+    from routes import (browser, dev, factcheck, place, placelist, ranking,
+                        search, static, tools, topic_page)
     app.register_blueprint(browser.bp)
     app.register_blueprint(dev.bp)
```

Comment:
should dev, placelist and topic_page go to base_dc?

Reply:
i wasn't sure.. we can move it.